spec/spec_helper.rb

```diff
@@ -19,6 +19,8 @@
 require 'database_cleaner'
 require 'ffaker'
 
+EasyPost.api_key = 'CvzYtuda6KRI9JjG7SAHbA'
```

Comment:
Not entirely sure why this is needed in the spec; any external calls should be stubbed or handled with `vcr`.  Let me know, I may be off here :)